ffi/type.cpp

```diff
@@ -144,22 +144,7 @@ API_EXPORT(uint64_t)
 LLVMPY_GetTypeBitWidth(LLVMTypeRef type) {
     llvm::Type *unwrapped = llvm::unwrap(type);
     auto size = unwrapped->getPrimitiveSizeInBits();
-    return size.getFixedSize();
-}
-
-// FIXME: Remove me once typed pointers support is removed.
-API_EXPORT(LLVMTypeRef)
-LLVMPY_GetElementType(LLVMTypeRef type) {
-    llvm::Type *unwrapped = llvm::unwrap(type);
-    llvm::PointerType *ty = llvm::dyn_cast<llvm::PointerType>(unwrapped);
-    if (ty != nullptr) {
-#if LLVM_VERSION_MAJOR < 14
-        return llvm::wrap(ty->getElementType());
-#else
-        return llvm::wrap(ty->getPointerElementType());
-#endif
-    }
-    return nullptr;
+    return size.getFixedValue();
```

Comment:
Note on this change: `getFixedSize()` retured `getFixedValue()` anyway, and `getFixedSize()` is deprecated / removed in later LLVM versions.

Reply:
Yep, I believe these are just API renames.